spec/system/managing_collections_spec.rb

```diff
@@ -0,0 +1,25 @@
+require 'rails_helper'
+
+RSpec.describe 'managing collections' do
+  let!(:collection) { FactoryBot.create(:collection, name: 'dogs and cats') }
+  let!(:topic){ FactoryBot.create(:topic, name: 'dog topic') }
+  let!(:unrelated_topic){ FactoryBot.create(:topic, name: 'unrelated') }
```

Comment:
_:unrelated_topic_ - nice. Very declarative and intention revealing.

Reply:
haha... this was the test you wrote for me ;)